templates/calico.go

```diff
@@ -515,4 +515,52 @@ spec:
         - key: aws-ippool
           path: aws-ippool.yaml
           {{end}}
+{{if eq .CloudProvider "gce"}}
+## gce stuff here
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: gce-ippool
+  namespace: kube-system
+data:
+  gce-ippool: |-
```

Comment:
Since the ippool is very similar to aws pool, you can do something like {{ .CloudProvider }}-ippool and use one manifest for config map and one calicoctl pod